relay-general/src/protocol/device_class.rs

```diff
@@ -99,5 +100,88 @@ static APPLE_DEVICE_MODEL_TO_CLASS_MAP: Lazy<BTreeMap<&str, DeviceClass>> = Lazy
         ("iPhone14,8", DeviceClass::HIGH),
         ("iPhone15,2", DeviceClass::HIGH),
         ("iPhone15,3", DeviceClass::HIGH),
+        // iPads
+        ("iPad1,1", DeviceClass::LOW),
+        ("iPad1,2", DeviceClass::LOW),
+        ("iPad2,1", DeviceClass::LOW),
+        ("iPad2,2", DeviceClass::LOW),
+        ("iPad2,3", DeviceClass::LOW),
+        ("iPad2,4", DeviceClass::LOW),
+        ("iPad3,1", DeviceClass::LOW),
+        ("iPad3,2", DeviceClass::LOW),
+        ("iPad3,3", DeviceClass::LOW),
+        ("iPad2,5", DeviceClass::LOW),
+        ("iPad2,6", DeviceClass::LOW),
+        ("iPad2,7", DeviceClass::LOW),
+        ("iPad3,4", DeviceClass::LOW),
+        ("iPad3,5", DeviceClass::LOW),
+        ("iPad3,6", DeviceClass::LOW),
+        ("iPad4,1", DeviceClass::LOW),
+        ("iPad4,2", DeviceClass::LOW),
+        ("iPad4,3", DeviceClass::LOW),
+        ("iPad4,4", DeviceClass::LOW),
+        ("iPad4,5", DeviceClass::LOW),
+        ("iPad4,6", DeviceClass::LOW),
+        ("iPad4,7", DeviceClass::LOW),
+        ("iPad4,8", DeviceClass::LOW),
+        ("iPad4,9", DeviceClass::LOW),
+        ("iPad5,1", DeviceClass::LOW),
+        ("iPad5,2", DeviceClass::LOW),
+        ("iPad5,3", DeviceClass::LOW),
+        ("iPad5,4", DeviceClass::LOW),
+        ("iPad6,3", DeviceClass::MEDIUM),
+        ("iPad6,4", DeviceClass::MEDIUM),
+        ("iPad6,7", DeviceClass::MEDIUM),
+        ("iPad6,8", DeviceClass::MEDIUM),
+        ("iPad6,11", DeviceClass::LOW),
+        ("iPad6,12", DeviceClass::LOW),
```

Comment:
These two lows are intentional? Or the mediums above?

Reply:
Yup this is intentional! The 4 mediums above these two are 1st gen iPad pros which have significantly faster processors compared to these two which are regular 5th gen iPads.

Thanks for the review!